### ISSUE TITLE
Errors when fine-tuning

### ISSUE THREAD

**Issue by tensorboy** [haotian-liu/LLaVA#79]
```
    File "/mnt/bd/data-tns-algo-masp-llm/experiment/LLaVA/llava/train/train.py", line 567, in train
      trainer.train()
    File "/mnt/bd/data-tns-algo-masp-llm/environment/anaconda3/envs/caption/lib/python3.9/site-packages/transformers/trainer.py", line 1644, in train
      return inner_training_loop(
    File "/mnt/bd/data-tns-algo-masp-llm/environment/anaconda3/envs/caption/lib/python3.9/site-packages/transformers/trainer.py", line 1731, in _inner_training_loop
      model = self._wrap_model(self.model_wrapped)
    File "/mnt/bd/data-tns-algo-masp-llm/environment/anaconda3/envs/caption/lib/python3.9/site-packages/transformers/trainer.py", line 1469, in _wrap_model
      self.model = model = FSDP(
  TypeError: __init__() got an unexpected keyword argument 'forward_prefetch'
```

seems it needs downgrade transformers to 4.26.1:
https://github.com/huggingface/transformers/issues/22446

**Comment by haotian-liu** [comment]
Hi, we use PyTorch 2.0 for developing the code, and it seems that this is an argument newly introduced in later PyTorch versions, which is addressed in this PR: https://github.com/huggingface/transformers/pull/22489. I will take a look into this later on the compatibility issue of upgrading the `transformers` package.

As a solution, I would suggest upgrading the PyTorch to 2.0, instead of downgrading `transformers`, as downgrading `trasnformers` may give rise to other incompatibility issues.

Btw, maybe this can solve the `torchrun` with 8-GPU issue as well?

**Comment by tensorboy** [comment]
> Hi, we use PyTorch 2.0 for developing the code, and it seems that this is an argument newly introduced in later PyTorch versions, which is addressed in this PR: [huggingface/transformers#22489](https://github.com/huggingface/transformers/pull/22489). I will take a look into this later on the compatibility issue of upgrading the `transformers` package.
> 
> As a solution, I would suggest upgrading the PyTorch to 2.0, instead of downgrading `transformers`, as downgrading `trasnformers` may give rise to other incompatibility issues.
> 
> Btw, maybe this can solve the `torchrun` with 8-GPU issue as well?

I'm using pytorch 1.12.1 now, I will upgrade to pytorch 2.0 and have a try, will let you know if it can work with 8-GPUs. 

Thanks for the quick feedbacks!!!

**Comment by sameeravithana** [comment]
Is it working with pytorch 1.12.1? I am still facing the issue. The problem is that I can not update torch due to dependency in the cuda drivers 11.4.
